src/App/App.scss

```diff
@@ -6,4 +6,12 @@
 
 .App {
   text-align: center;
+
+  main {
+    position: absolute;
```

Comment:
I think we will still need to figure out a way so that when the screen is made larger and smaller that the large palette plus sign scales vertically with the page so the bottom palette can always be seen. But much closer than before!